packages/compiler-core/src/transforms/transformSlotOutlet.ts

```diff
@@ -68,9 +69,22 @@ export function processSlotOutlet(
   const propsWithoutName = name
     ? node.props.filter(p => p !== name)
     : node.props
+
   if (propsWithoutName.length > 0) {
+    //#2488
+    propsWithoutName.forEach(prop => {
+      if (
+        prop.type === NodeTypes.DIRECTIVE &&
```

Comment:
Note: here only `v-bind` args should be camelized so we should check `prop.name === 'bind'`. I also refactored the logic to reduce iterations from 3 to 1, see 7cd85be2